packages/openapi-generator/src/generator.ts

```diff
@@ -20,6 +27,10 @@ const logger = createLogger('openapi-generator');
  * @param options Options to configure generation.
  */
 export async function generate(options: GeneratorOptions): Promise<void> {
+  options.serviceMapping =
```

Comment:
I would not adjust the original object but use a variable which the path to the file which is then passed to readServiceMapping. Or even do the fallback path inside the readServiceMapping method?